manifests/init.pp

```diff
@@ -9,6 +9,7 @@
   package {'ssh':
     ensure => present,
     name   => $package_name,
+    notify => Service['ssh'],
```

Comment:
Since there's already a relationship between the resources, I'd change the `require` to a `subscribe` below in the `Service['ssh']` resource.

Reply:
Thank you for your suggestion. Done!